samples/snippets/async_api.py

```diff
@@ -0,0 +1,36 @@
+# [START cloudoptimization_async_api]
+
+import json
+import sys
+
+from google.api_core.exceptions import GoogleAPICallError
+from google.cloud.optimization_v1.services.fleet_routing.client import FleetRoutingClient
+from google.cloud.optimization_v1.types.fleet_routing import BatchOptimizeToursRequest
+
+
+def call_async_api(request_file_name):
+  """Call the async api for fleet routing."""
+  # Use the default credentials for the environment to authenticate the client.
+  fleet_routing_client = FleetRoutingClient()
+
+  with open(request_file_name, 'r') as f:
+    # The request must include the "parent" field with the value set to
+    # "projects/{YOUR_GCP_PROJECT_ID}".
```

Comment:
The `project_id` is best to be set in `resources/sync_request.json`. I can make it an argument and let the function inject it into the request, but it might be too cumbersome for our users in real life use case. I can avoid explicitly calling out the variables in the comment. For example, I can change the comment to 
```
Please replace the variables in "resources/sync_request.json" before calling the API.
```
Please let me know if that sounds good.